src/Microsoft.Extensions.CommandLineUtils/Microsoft.Extensions.CommandLineUtils.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <Description>Command-line parsing API. Commonly used types:
+Microsoft.Extensions.CommandLineUtils.CommandLineApplication
```

Comment:
Love the newlines! 😄 Do they end up showing up correctly in the package's nuspec?